src/coreclr/jit/llvm.cpp

```diff
@@ -1555,7 +1593,11 @@ Value* Llvm::localVar(GenTreeLclVar* lclVar)
     Value*       llvmRef;
     unsigned int lclNum = lclVar->GetLclNum();
     unsigned int ssaNum = lclVar->GetSsaNum();
-    if (_localsMap->find({lclNum, ssaNum}) == _localsMap->end())
+    if (m_allocas[lclNum] != nullptr)
```

Comment:
```suggestion
    if (isLlvmFrameLocal(varDsc))
```
(`varDsc` definition will need to be moved to before the `if`)